src/Eloquent/Model.php

```diff
@@ -502,19 +570,6 @@ protected function isGuardableColumn($key)
         return true;
     }
 
-    /**
-     * @inheritdoc
-     */
-    public function __call($method, $parameters)
```

Comment:
It's nice to see this gone! :)

Reply:
I'm not sure why the method `unset` was not declared. It might be to allow a relation named `unset`.